### ISSUE TITLE
"Parser error on XML" Samsung 360 gear video

### ISSUE THREAD

**Issue by Thyvault** [google/spatial-media#172]
![parser error](https://user-images.githubusercontent.com/29265454/26906339-32272a04-4bb2-11e7-9f71-e320eb1c3dcb.jpg)

Hello everyone. I'm getting this error when tryin to inject metadata to a samsung 360 gear video, mp4. What am I doing wrong?

Thank you very much.

**Comment by Pablos3331** [comment]
Have the same. One issue earlier :-)

**Comment by wesleytansg** [comment]
To both Thyvault and Pablos3331:

You guys should not have to use the metadata injector for videos produced using the Gear 360.  As long as you import the clips into Action Director you have to then export it.  It is not enough to just stitch them in Action Director which will create a equirectangular mapping video file.  I had the same problem but the moment I exported the video clip using Action Director, the clip can then be uploaded to Youtube straightaway.  Action Director injects the metadata for the 360 video when you are exporting the file.

Good luck.

**Comment by acolwell** [comment]
Could you please attach a small example file that exposes this bug. Eventhough a workaround has been mentioned, I'd like to see what is going wrong here. Thanks.

**Comment by netjeff** [comment]
I'm having the same symptoms.  I've attached a short video file that has the symptoms.  

Shot on my Gear 360 (model SM-C200) running firmware C200GLU0AQF1, then copied from microSD card onto my Win10 PC.  Then in Gear 360 ActionDirector v2.0.1619.0 I added the file into a temporary project, and AD automatically stitched into the attached file.  I **not** use the "Produce" button, I simply found the stitched file and tried to use the latest Spatial Media Metadata Injector v2.0 on that file.
[360_0092_Stitch_XHC.zip](https://github.com/google/spatial-media/files/1255104/360_0092_Stitch_XHC.zip)
